.github/workflows/tests.yml

```diff
@@ -62,3 +64,8 @@ jobs:
         run: |
           sudo apt update && sudo apt install gcc-multilib
           cargo test $TEST_MODE -p openmls --verbose --target i686-unknown-linux-gnu
+
+      - name: Tests Wasm
```

Comment:
Could you move all the wasm tests in a separate job ? So that it can run independently from the rest of the tests and not slow them down ?